spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinProperties.java

```diff
@@ -33,6 +33,7 @@
 	private String baseUrl = "http://localhost:9411/";
 	private boolean enabled = true;
 	private int flushInterval = 1;
+	private boolean thrift = false;
```

Comment:
ex `private Encoding encoding = Encoding.JSON;` assuming property-fying works as it should